lms/djangoapps/commerce/utils.py

```diff
@@ -100,9 +100,11 @@ def get_checkout_page_url(self, *skus):
         Example:
             http://localhost:8002/basket/add/?sku=5H3HG5&sku=57FHHD
         """
-        return '{checkout_page_path}?{skus}'.format(
+        program_uuid = kwargs.get('program_uuid')
+        return '{checkout_page_path}?{skus}&{bundled_program}'.format(
             checkout_page_path=self.get_absolute_ecommerce_url(self.config.MULTIPLE_ITEMS_BASKET_PAGE_URL),
             skus=urlencode({'sku': skus}, doseq=True),
+            bundled_program=urlencode({'bundle':program_uuid}) if program_uuid else ''
```

Comment:
This could leave you with a dangling `&`. How about:
```
url = '{checkout_page_path}?{skus}'.format(
    checkout_page_path=self.get_absolute_ecommerce_url(self.config.MULTIPLE_ITEMS_BASKET_PAGE_URL),
    skus=urlencode({'sku': skus}, doseq=True),
)
if program_uuid:
    url = '{url}&bundle={program_uuid}'.format(
        url=url,
        program_uuid=program_uuid,
    )
    return url
```